packages/eslint-plugin-mdx/src/processors/remark.ts

```diff
@@ -30,27 +30,29 @@ export const remark: Linter.Processor = {
     return [
       ...mdxMessages,
       ...processors.markdown.postprocess(markdownMessages, filename),
-    ].map(lintMessage => {
-      const {
-        message,
-        ruleId: eslintRuleId,
-        severity: eslintSeverity,
-      } = lintMessage
+    ]
+      .sort((a, b) => a.line - b.line)
```

Comment:
```suggestion
      .sort((a, b) => a.line - b.line || a.column - b.column)
```

In case for same lines?